src/store/getters.ts

```diff
@@ -62,14 +77,24 @@ export default {
     return state.authRoles.includes('view')
   },
 
-  /** Returns whether business is in good standing. */
+  /** Is True if business is in good standing. */
   isInGoodStanding (state: StateIF): boolean {
-    return state.entityStatus === EntityStatus.GOOD_STANDING
+    return (state.entityStatus === EntityStatus.GOOD_STANDING)
+  },
+
+  /** Is True if business is pending dissolution. */
+  isPendingDissolution (state: StateIF): boolean {
+    return (state.entityStatus === EntityStatus.PENDING_DISSOLUTION)
+  },
+
+  /** Is True if business is not in compliance. */
+  isNotInCompliance (state: StateIF): boolean {
+    return (state.entityStatus === EntityStatus.NOT_IN_COMPLIANCE)
   },
 
   nrNumber (state: StateIF): string {
```

Comment:
a lot of properties are prefixed with `get` in this getters file.  Do we want to update the other ones to be prefixed with `get`?  i.e. nrNumber, reportState, entityIncNo, coaEffectiveDate

Reply:
I tried to add some consistency to the getters, as you mention, but it's not perfect. For example, we have a lot of getters with the name `isXxx..`. Should these be changed to `getIsXxx..`? I don't love that. And either way, getters could be confused with local (component) getters or properties. So the current style is not perfect.

That said, there are some low hanging fruit (the ones you mention) that I will change over :)

Reply:
Done.